migrations/versions/24da7df1e00e_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: 24da7df1e00e
+Revises: 52d175c3da98
+Create Date: 2017-06-27 16:32:54.671408
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '24da7df1e00e'
+down_revision = '52d175c3da98'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'event_topics', ['id'])
+    op.create_unique_constraint(None, 'event_types', ['id'])
```

Comment:
> module level import not at top of file

For the above, manually change the migration file for now, change `script.py.mako` also so that this message doesn't occur in future